h2o-docs/src/product/api-changes.rst

```diff
@@ -3,6 +3,12 @@ API-Related Changes
 
 H2O-3 does its best to keep backwards compatibility between major versions, but sometimes breaking changes are needed in order to improve code quality and to address issues. This section provides a list of current breaking changes between specific releases.
 
+From 3.30.0.5
+~~~~~~~~~~~~~
+
+The ``h2o-scala`` module is deprecated in version 3.30.0.5 and will be completely removed in the next major version, 3.32.0.1. This by no means has any impact on Scala support, as the ``h2o-scala`` module was intended for internal use only and does not represent the H2O Scala API.
```

Comment:
> as the ``h2o-scala`` module was intended for internal use only and does not represent the H2O Scala API

This is not correct, the module was actually intended for public use, we even made examples how to use it independent of Sparkling Water: https://github.com/h2oai/h2o-droplets/blob/master/h2o-scala-droplet/

Specifically 
https://github.com/h2oai/h2o-droplets/blob/master/h2o-scala-droplet/build.gradle#L84